smoke-tests/designer/features/actions/actions.js

```diff
@@ -1,4 +1,4 @@
-const addComponent = require("../pageobjects/pages/addComponent.page");
+const createComponent = require("../pageobjects/sections/createComponent.section");
 const configPage = require("../pageobjects/pages/config.page");
 const formDesigner = require("../pageobjects/pages/formDesigner.page");
```

Comment:
a common js pattern is to in the pageobjects/pages directory, add an index.js, which imports and reexports all/most the things you need - no need to action unless you want to 

Then you can just do `const { addComponent, configPage, formDesigner } = require('../pageobjects/pages') //no /index needed`

Reply:
@jenbutongit great suggestion and implemented